From f5a951bd6cd14ddb27ee0be1835e891e1868b350 Mon Sep 17 00:00:00 2001
From: Michael Butler <butlermichael@google.com>
Date: Wed, 19 May 2021 00:13:39 +0000
Subject: [PATCH 1/3] Revert "Add new llndk library, libneuralnetworks_shim.so"

This reverts commit 8685248a99095f3eef009fad3b44a1ad5941ba78.

Reason for revert: Removing libneuralnetworks_shim.so from Android S

Change-Id: I3fa3092e02e9d1bbf00d801ad68ea754bfbd8c37
---
 target/product/gsi/current.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/gsi/current.txt b/target/product/gsi/current.txt
index 550ae7c315..c753e6c3be 100644
--- a/target/product/gsi/current.txt
+++ b/target/product/gsi/current.txt
@@ -14,7 +14,6 @@ LLNDK: libm.so
 LLNDK: libmediandk.so
 LLNDK: libnativewindow.so
 LLNDK: libneuralnetworks.so
-LLNDK: libneuralnetworks_shim.so
 LLNDK: libselinux.so
 LLNDK: libsync.so
 LLNDK: libvndksupport.so
-- 
2.17.1

